### PR TITLE
3 client side route for food list items and add links

### LINKED ISSUE #3: Create client-side route for individual food + add Links to FoodList items
We need a client-side route to show the `FoodDetails` component (talk to your lead developer if this component hasn't been created yet!).

Create a route in the `App` component to render the `FoodDetails` component at `/details/:foodId`.

Relates to #4

### PR DESCRIPTION
**Pull request by raaya-pathare**
Closes #3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,6 +66,7 @@
     "npm-run-all": "^4.1.5",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "redux-mock-store": "^1.5.3",
```
Comment (emilyparkes) [inline package.json:69]: Who did you get this change approved by? There is no mention of this addition in your PR comments. 
Thanks! :)

File: tests/client/components/FoodList.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+// import { shallow } from 'enzyme'
```
Comment (emilyparkes) [inline tests/client/components/FoodList.test.js]: No commented out statements please.

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: client/components/App.jsx [hunk 3]
```diff
@@ -5,16 +5,18 @@ import { Container } from 'semantic-ui-react'
...
   return (
     <Router>
       <>
         <Route path='/' component={TopMenu} />
-        <Container style={{ paddingTop: 75 }}>
+        <Container style={{ marginTop: 75 }}>
           <Switch>
             <Route exact path='/' component={FoodList} />
             <Route component={NoMatch} />
+            <Route path='/details/:foodId' component={FoodDetails} />
```
Comment (emilyparkes) [inline client/components/App.jsx:19]: Please also add a link to each food (in FoodList) that will take us to `/details/:foodId`. Thanks.

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (raaya-pathare) [reply package.json:69]: We got approval from Bryce to install this package in order to write a functioning test! :)

Comment (emilyparkes) [reply package.json:69]: Sweet, thank you!

**Review by emilyparkes — APPROVED** [review]

**Comment by bryceknz** [comment]
Why was this merged into `development` with an empty `EditFood.jsx` file? That doesn't seem to be related to issue #3

**Pull request merged**